CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# 7.1.2 (2020-08-05)
+
+### Features / Enhancements
+  * **Explore**: Don't run queries on datasource change. [#26033](https://github.com/grafana/grafana/pull/26033), [@davkal](https://github.com/davkal)
+  * **TemplateSrv**: Formatting options for ${__from} and ${__to}, unix seconds epoch, ISO 8601/RFC 3339. [#26466](https://github.com/grafana/grafana/pull/26466), [@torkelo](https://github.com/torkelo)
+  * **Toolkit/Plugin**: throw an Error instead of a string. [#26618](https://github.com/grafana/grafana/pull/26618), [@leventebalogh](https://github.com/leventebalogh)
+
+  ### Bug Fixes
+  * **Dashbard**: Fix refresh interval settings to allow setting it to equal min_refresh_interval. [#26615](https://github.com/grafana/grafana/pull/26615), [@torkelo](https://github.com/torkelo)
+  * **Explore**: Run queries on Metrics to Logs transition. [#26704](https://github.com/grafana/grafana/pull/26704), [@ivanahuckova](https://github.com/ivanahuckova)
```

Comment:
```suggestion
```

I would remove this as the bug is introduced and fixed in the same release - therefore the end users don't have to be aware of it/it has no impact on them.